hash/pc8801_flop.xml

```diff
@@ -48093,21 +48557,31 @@ loderuna - pc8801       Device Floppy Disk load failed: Invalid image
 		</part>
 	</software>
 
-	<software name="kuryutos">
-		<description>Kuryuto(SR ver)</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+	<!-- Hangs at "Starcraft presents" screen -->
+	<!-- FDC stalls [Tries to match CHRN=(1b 00 01 02) and CHRN=(1b 00 02 01) with (1b 00 01 03)] -->
+	<software name="kuryutos" supported="no">
+		<description>Kyu Ryu Tou (SR ver)</description>
```

Comment:
The official Latin script title appears to be written without spaces, as “KyuRyuTou”, e.g. see the screenshot here: https://www.gamepres.org/pc88/library/1986/KYU00.gif

Reply:
Yep, saw that since that's how I've identified the game. Wasn't sure how to treat spacing but okay.
![0134](https://user-images.githubusercontent.com/9251271/132139214-b1325a2a-0c21-4fca-b57b-d02aebdf3cc5.png)
I've also read the story premise of this, it's actually really referring to Kowloon Island as the kanji title suggests.